WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewController.swift

```diff
@@ -42,7 +42,7 @@ final class ShippingLabelAddressFormViewController: UIViewController {
             actionSheet.view.tintColor = .text
 
             actionSheet.addCancelActionWithTitle(Localization.contactActionCancel)
-            if let email = email, email.isNotEmpty {
+            if let email = email, email.isNotEmpty && MFMailComposeViewController.canSendMail() {
```

Comment:
nit: You can use `if let email` now (as of Xcode 14), instead of `if let email = email`.